pkg/server/controllers.go

```diff
@@ -621,6 +621,52 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 		return err
 	}
 
+	controllerName = "permission-claim-label-controller"
+	if err := server.AddPostStartHook(postStartHookName(controllerName), func(hookContext genericapiserver.PostStartHookContext) error {
+		logger := klog.FromContext(ctx).WithValues("postStartHook", postStartHookName(controllerName))
+		// do custom wait logic here because APIExports+APIBindings are special as system CRDs,
```

Comment:
Don't think we want to copy/paste this. These controllers should be able to use the normal `s.waitForSync` call, I think.

Reply:
Also will need to remove starting the 2 permission claim controllers from L616-617